model/reward/instructor/requirements.txt

```diff
@@ -0,0 +1,140 @@
+aiohttp==3.8.3
+aiosignal==1.3.1
```

Comment:
This is a pretty extensive requirements.txt. Are all these libraries really required for the training of the model?

Reply:
was already fixed by @theblackcat102